models/articles.models.js

```diff
@@ -1,20 +1,33 @@
 const db=require("../db/connection.js")
 
-exports.selectAllArticles=((articleQuery)=>{
-    //articleQuery= "articles." + articleQuery
-    //console.log(articleQuery)
+exports.selectAllArticles=((articleQuery, articleSortType)=>{
+    const allowedInputs=["title", "topic", "author", "body", "created_at", "ASC", "DESC", "votes", undefined]
 
-    return db.query(`SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.comment_id)::INT AS comment_count
-    FROM articles
-    INNER JOIN comments
-    ON articles.article_id =comments.article_id
-    GROUP BY articles.article_id
-    ORDER BY created_at DESC
-    ;`)
+    if(articleQuery===undefined)articleQuery="created_at"
```

Comment:
Setting this as a default in your parameters would also mean you could get rid of this line.